css/index.css

```diff
@@ -48,30 +48,36 @@ h2 {
 /*start of my flexbox container for experience items */
 .exp {
     display: flex;
-    flex-direction: column; /* will stack items vertically */
+    flex-direction: column;
+    /* will stack items vertically */
     text-align: center;
 }
 
 /* each experience item container*/
 .exp-item {
     display: flex;
     justify-content: space-between;
-    align-items: center; /* centers the content vertically */
+    /* centers the content vertically */
```

Comment:
suggest using flex-direction: row - and thus the spacing between title and date will seem more natural as display grows or shrinks
column-gap:5em would give you separation between title and date